packages/console/src/consts/page-tabs.ts

```diff
@@ -0,0 +1,5 @@
+export const signInExperiencePage = {
+  brandingTab: 'branding',
+  signUpAndSignInTab: 'sign-up-and-sign-in',
+  othersTab: 'others',
+};
```

Comment:
nit: enum?

Reply:
or object.freeze so it may has type guard

Reply:
updated with enum